src/client/components/pages/search.js

```diff
@@ -83,7 +83,7 @@ class SearchPage extends React.Component {
 			.then((data) => {
 				this.setState(prevState => ({
 					nextEnabled: data.length >= prevState.size,
-					results: data
+					results: data.filter(Boolean)
```

Comment:
This reads a little confusing. I'd like it better if there were a function to explicitly check expected values rather than relying on what I find to be kind of unintuitive behaviors of the `Boolean` ctor in JS.

Reply:
Fair enough, I'll get that replaced.